content/en-US/api/ScreenAdvanced.md

```diff
@@ -0,0 +1,96 @@
+---
+recipe: api-interface
+title: 'ScreenAdvanced'
+mdn_url: /en-US/docs/Web/API/ScreenAdvanced
+specifications: https://webscreens.github.io/window-placement/#screenadvanced
+browser_compatibility: api.ScreenAdvanced
+---
+
+## Description
+
+The `ScreenAdvanced` interface of the Window Placement API extends the
+`Screen` interface and provides additional per-screen information.
+
```

Comment:
```suggestion

The `ScreenAdvanced` interface is a child of this one in inherits its members.
```

Reply:
That's backwards though, isn't it? This page describes `ScreenAdvanced`

Reply:
I did the rewrite wrong. Please look again.

Reply:
PTAL

Reply:
Ping you again. It would nice to have our first ever puppy doc up.